js-intro.js

```diff
@@ -4,39 +4,63 @@
 // Please read all questions thoroughly
 // Psuedo coding is highly recommended
 
-// --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
+// -----COMPLETE-----1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
-
-
-// --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
+// create a function that determines if a given temp is below boiling point (bp<212), at boiling point (=212), or above boiling point (bp>212)
+var boilingPoint = (temp) => {
+    if (temp<212) {
+        return (`${temp} is below boiling point`)
+    } else if (temp>212) {
+        return (`${temp} is above boiling point`)
+    } else{
+        return (`${temp} is at boiling point`)
+    }
+}
+
+console.log(boilingPoint(temp1));
+console.log(boilingPoint(temp2));
+console.log(boilingPoint(temp3));
+
+// ----COMPLETE------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
-var myNumbers1 = [3, 7, 0, 6, -9]
-
+//create function that multiplies each number in array by 5 using a for loop
 
+var myNumbers1 = [3, 7, 0, 6, -9]
 
+for(let i=0; i<myNumbers1.length; i++) {
+    console.log(myNumbers1[i] *5)
+}
```

Comment:
You're on the right track here! Consider using a new variable with an empty array to house the new values * 5. Also this just needs to be wrapped in a function to make it reusable :) In addition, `console.log()` should be reserved for de-bugging, etc. A `return` would be more appropriate.